pom.xml

```diff
@@ -80,18 +78,18 @@
     <version.plugin.plexus-component-metadata>2.1.0</version.plugin.plexus-component-metadata>
     <version.plugin.sisu>0.3.4</version.plugin.sisu>
     <version.plugin.plugin>3.6.0</version.plugin.plugin>
-    <version.plugin.release>3.0.0-M1</version.plugin.release>
-    <version.plugin.resources>3.1.0</version.plugin.resources>
+    <version.plugin.release>3.0.0-M4</version.plugin.release>
+    <version.plugin.resources>3.2.0</version.plugin.resources>
     <version.plugin.site>3.9.1</version.plugin.site>
     <version.plugin.source>3.2.1</version.plugin.source>
     <version.plugin.surefire>3.0.0-M5</version.plugin.surefire>
     <version.plugin.pmd>3.13.0</version.plugin.pmd>
-    <version.plugin.license>3.0</version.plugin.license>
+    <version.plugin.license>4.1</version.plugin.license>
 
     <enforceManagedDeps>true</enforceManagedDeps>
 
     <unEnforcedPlugins>org.eclipse.m2e:lifecycle-mapping</unEnforcedPlugins>
-    <enforcedMavenVersion>[3.2.5,)</enforcedMavenVersion>
+    <enforcedMavenVersion>[3.3.9,)</enforcedMavenVersion>
```

Comment:
I don't know exactly why you chose those specific version upgrades, but they look fine.

Reply:
3.3.9 was the minimum required by certain plugns so I just did a version bump